examples/any-type/any_type.bal

```diff
@@ -1,6 +1,6 @@
 import ballerina/io;
 
-// A variable of type `any` can hold any value except an error.
+// A variable of type `any` can hold any value except an `error`.
```

Comment:
If we are emphasizing it we are referring to the error type. IMO, we should then say

```suggestion
// A variable of type `any` can hold any value except an `error` value.
```